packages/apps/transfer/src/constants/kadena.ts

```diff
@@ -0,0 +1,35 @@
+export const kadenaConstants = {
+  MAINNET: {
+    API: process.env.KADENA_MAINNET_API ?? '',
+    NETWORKS: {
+      MAINNET01: 'mainnet01',
+    },
+    API_HOST: ({
+      networkId,
+      chainId,
+    }: {
+      networkId: string;
+      chainId: string;
+    }) =>
+      `${process.env.KADENA_MAINNET_API}/chainweb/0.0/${networkId}/chain/${chainId}/pact` ??
```

Comment:
I usually use a util function that retrieves all the env vars, adds typings and does parsing where needed. This also allows for easy tracking of all env-vars.

E.g. https://github.com/kadena-community/kadena.js/blob/master/packages/apps/graph/src/utils/dotenv.ts

Reply:
Oh, nvm :) I see this is that file 😅

Reply:
added a new layer for env with an interface and a function to get envs 
the only issue is it can't be dynamically following the docs from nextjs they replace process.env.* with their value so it's not a normal javascript object so it can't be called dynamically like process.env[key] so I had to map it to an actual object and then get what I wanted